mmv1/products/accesscontextmanager/api.yaml

```diff
@@ -1,2113 +0,0 @@
-# Copyright 2018 Google Inc.
```

Comment:
As a follow-up would it be easy to sort the `provider.go` resources, so that we don't see diffs on future changes? I _think_ the diff is because ordering in the API.yaml file determines the order in `provider.go` right now, and this change makes it reverse alphabetical (by product). If we sorted by alphabetical we'd match the order on the filesystem, and could normalize across both the new + old format.

Reply:
I can take a look at making a separate sorting change